OpenRA.Game/Map/MapGrid.cs

```diff
@@ -123,7 +123,7 @@ public class MapGrid : IGlobalModData
 
 		public CellRamp[] Ramps { get; }
 
-		internal readonly CVec[][] TilesByDistance;
+		public CVec[][] TilesByDistance { get; private set; }
```

Comment:
```suggestion
		public readonly CVec[][] TilesByDistance;
```

?

Reply:
Both seem to be equivalent but sure, has been updated.